clients/web/src/templates/widgets/taskProgress.pug

```diff
@@ -6,6 +6,11 @@
           = percentText
         .progress-left
           = timeLeftText
-  .g-task-progress-title(title=progress.data.title)= progress.data.title
+  .g-task-progress-title(title=progress.data.title)
+    if progress.data.resource && progress.data.resourceName
```

Comment:
Since in the test you demonstrate having the resource the simple ID rather than an object with an _id element, perhaps this test should be `if progress.data.resource && progress.data.resource._id && progress.data.resourceName`.